src/test/java/formflow/library/inputs/RequiredInputs.java

```diff
@@ -0,0 +1,47 @@
+package formflow.library.inputs;
+
+import formflow.library.data.FlowInputs;
```

Comment:
Do you want to take this test code out (along with the related flows-config & template files)?